web/src/input_pill.ts

```diff
@@ -28,7 +29,7 @@ type InputPillCreateOptions<ItemType> = {
         existing_items: ItemType[],
         pill_config?: InputPillConfig,
     ) => ItemType | undefined;
-    get_text_from_item: (item: ItemType) => string;
+    get_text_from_item: (item: ItemType) => string | Invitees;
```

Comment:
`get_text_from_item` is only used for copying, so this should just return a string always I guess. I think it can just return a string including both email and name or something like that.